Launching-new-features.md

```diff
@@ -27,7 +27,7 @@ Say you are working on a large scale user-facing feature that will take 1 or mor
 
 4. The very last PR you make (to finish up the feature you are working on) must include changes that move the feature flag to the TEST stage. This is to ensure that the feature is available in the test environment, so that we can feature-test it before it is made available to the users in the production environment. **NOTE: Please test all the changes manually to make sure that the feature works fully end-to-end on your local dev server, before moving the flag to the TEST stage.**
 
-5. When the PR is merged, fill out [this form](https://forms.gle/zDCsoN6Xb6JvQku87) to request that your feature be tested. This form will be processed by the server admins team, who will work with you and the feature testers to set a date during which your feature will be available to use on one of our test servers. (We can only surface the feature for a limited time because the servers are also needed for other things, such as release testing.) Once the testing date is confirmed, the server admin will make a deployment, turn the feature flag on, and send instructions to the feature testers for how to test the feature. Feature testers will use this [feature review template](https://docs.google.com/document/d/1ibQC9t1jnOg-o1lrHEM3QEXeF4eSqAiPhs2lD2iCpy4/edit) to conduct the testing.
+5. When your final PR is close to being merged, fill out [this form](https://forms.gle/zDCsoN6Xb6JvQku87) to request that your feature be tested. This form will be processed by the server admins team, who will work with you and the feature testers to set a date during which your feature will be available to use on one of our test servers. (We can only surface the feature for a limited time because the servers are also needed for other things, such as release testing.) Once the testing date is confirmed, the server admin will make a deployment, turn the feature flag on, and send instructions to the feature testers for how to test the feature. Feature testers will use this [feature review template](https://docs.google.com/document/d/1ibQC9t1jnOg-o1lrHEM3QEXeF4eSqAiPhs2lD2iCpy4/edit) to conduct the testing.
```

Comment:
It sounds a bit vague to me, maybe say when it's approved by all of the reviewers or is in merge queue ? And also why do we need to change it to  this, was there any issue with the former one ?

Reply:
@Ash-2k3 This actually came about because of your "undo" PR :) When you asked me about filling the form, I went back and checked with the server admins if it would be useful to have the notification earlier, and they said it would be. So I feel like, as long as we know the PR is going to be merged "soon" with no major modifications, filling in the form is fine (e.g. it's OK if a reviewer has a couple last straggling nit comments still).

Do you have a suggestion for how to convey this better?

Reply:
I see, an early notification makes sense then. But "close to being merged" is a bit unclear IMO -- It leaves room for interpretation about when exactly is the right time to fill out the form. I think "When your final PR has received all necessary approvals, .." would be better ?

Reply:
Or maybe "Once your PR has reached a stage where you don't anticipate major code changes, please fill out this form: --."

Reply:
Good idea, thanks. Done!